app/src/main/java/com/example/movopfy/uiComponents/navigation/BottomNavigationBar.kt

```diff
@@ -24,7 +24,7 @@ fun BottomNavigationBar(
 ) {
     NavigationBar(
         containerColor = BackgroundNavBar,
-        modifier = Modifier.height(MaterialTheme.size.height)
+        modifier = Modifier.height(MaterialTheme.size.heightComponent)
```

Comment:
strange name `heightComponent`

Reply:
and which name I should use? heightButton or just height?

Reply:
I did